package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "widget-common",
-  "version": "3.13.0",
+  "version": "3.13.1",
```

Comment:
@olegrise shouldn't this change ( new functionality ) require moving to 3.14.0 ?

Reply:
@santiagonoguez It's not a big change. @stulees what do you think?

Reply:
Given it's an additional _util_ function (helper) as opposed to a new module of functionality, I think it's just a minor update.

Reply:
good !